src/Components/SingleArticle.jsx

```diff
@@ -95,17 +76,14 @@ const SingleArticle = () => {
       <section className="article-content">
         <p>{article.body}</p>
       </section>
-      <div className="action-buttons">
-        <button className="vote-button">
-          <ArrowBigUp className="vote-icon-up" size={30}/>
-        </button>
-        <span className="vote-count">{article.votes}</span>
-        <button className="vote-button">
-          <ArrowBigDown className="vote-icon-down" size={30}/>
-        </button>
+      <div className="action-buttons sticky-actions">
+        <VoteButtons 
+          articleId={article_id}
+          initialVotes={article.votes}
+        />
         <button 
-          className={`action-button ${showComments ? 'active' : ''}`}
-          onClick={handleCommentsClick}
+          className={`action-button ${commentsVisible ? 'active' : ''}`} 
+          onClick={() => setCommentsVisible(!commentsVisible)}
```

Comment:
This could also be handled with React compostion - it would make the handling of state in the component cleaner. https://l2c.northcoders.com/courses/sd-notes/front-end#sectionId=composition-in-react,step=intro